python-lib/config.py

```diff
@@ -51,7 +52,12 @@ def get_table_name(dataset_config: Mapping[AnyStr, Any]) -> AnyStr:
         # e.g., mode == query
         raise ValueError(f'Snowflake connection mode {params["mode"]} is not currently supported.')
 
-    return f'"{params["schema"]}"."{params["table"]}"'.replace('${projectKey}', project_key)
```

Comment:
Below code looks much nicer imho. You do not need using `if` statement
`return f'"{params.get('schema', '')}"."{params["table"]}"'.replace('${projectKey}', project_key)`

Reply:
Again, you do need the if statement. See tests.